data/tools/pycodequal.yml

```diff
@@ -0,0 +1,10 @@
+name: PyCodeQual
+categories:
+  - linter
+tags:
+  - python
+license: proprietary
+types:
+  - service
+homepage: 'https://pycodequ.al'
+description: Static Python code analysis
```

Comment:
Hum, the description is rather generic right now. Do you think you can highlight a few key features that sets PyCodeQual apart from other services?
The [full list is here](https://analysis-tools.dev/tag/python) for reference.